### ISSUE TITLE
rslsync plugin no longer appears in Community Collection

### ISSUE THREAD

**Issue by brianannis** [ix-plugin-hub/iocage-plugin-index#127]
As of 11/17 rslsync no longer appears in the Community Collection view. Git clones of the plugin repo also tapered off at that time. No code changes have been made to the manifest, index, or plugin git repo since being added in #122 

<img width="833" alt="Screen Shot 2020-11-20 at 10 40 03 PM" src="https://user-images.githubusercontent.com/6808268/99866716-6928f700-2b81-11eb-8828-d31c9f2eb357.png">

I'm not sure how the middleware processes the plugin list, will look into logs on a TrueNAS box and see if there is anything interesting. But _something_ appears to be amiss.

**Comment by brianannis** [comment]
It appears that on my test TrueNAS 12-RELEASE server the box is currently polling `ix-plugin-hub/iocage-plugin-index:12.2-RELEASE`, which has not had changes from `master` merged in yet. Will changes from master be merged in for TrueNAS 12-U1 or 12.1-RELEASE?

Not 100% sure how the branching scheme works for plugins, but under the assumption that TrueNAS 12 would poll branch 12.0, 12.1, or 12.2 the plugin would have never appeared in the GUI. However it was available for several days after release based on my own testing and traffic to plugin repo. Was there a change somewhere to redirect the poll from `master` branch to `12.2-RELEASE` ?

```
root@truenas[~]# cd /mnt/tank/iocage/.plugins/github_com_ix-plugin-hub_iocage-plugin-index_git && git status
On branch 12.2-RELEASE
Your branch is up to date with 'origin/12.2-RELEASE'.

nothing to commit, working tree clean
```

**Comment by kmoore134** [comment]
We have to backport to the 12.2-RELEASE branch to make it live for 12, which was done a few moments ago.

**Comment by brianannis** [comment]
Fixed by #130

**Comment by brianannis** [comment]
Confirmed working, thank you!
